python/python/glide/async_commands/cluster_commands.py

```diff
@@ -514,3 +516,52 @@ async def flushall(
             TClusterResponse[TOK],
             await self._execute_command(RequestType.FlushAll, args, route),
         )
+
+    async def cluster_scan(
+        self,
+        cursor: ClusterScanCursor,
+        match: Optional[str] = None,
+        count: Optional[int] = None,
+        type: Optional[ObjectType] = None,
+    ) -> Tuple[ClusterScanCursor, List[str]]:
```

Comment:
Unfortunately, we don't support tuples yet - update return type to match the standard
```suggestion
    ) -> List[Union[str, List[str]]]:
```